dashboard/src/components/settings/permissions/PermissionVariableSettings/PermissionVariableSettings.tsx

```diff
@@ -58,32 +57,35 @@ export default function PermissionVariableSettings() {
     throw error;
   }
 
-  async function handleDeleteVariable({ key }: CustomClaim) {
-    const filteredCustomClaims = Object.keys(
-      data?.app?.authJwtCustomClaims,
-    ).filter((customClaimKey) => customClaimKey !== key);
-
-    const updateAppPromise = updateApp({
+  async function handleDeleteVariable({ id }: PermissionVariable) {
+    const updateConfigPromise = updateConfig({
       variables: {
-        id: currentApplication?.id,
-        app: {
-          authJwtCustomClaims: filteredCustomClaims.reduce(
-            (customClaims, currentKey) => ({
-              ...customClaims,
-              [currentKey]: data?.app?.authJwtCustomClaims[currentKey],
-            }),
-            {},
-          ),
+        appId: currentApplication?.id,
+        config: {
+          auth: {
+            session: {
+              accessToken: {
+                customClaims: permissionVariables
+                  ?.filter((permissionVariable) => permissionVariable.id !== id)
```

Comment:
the use of `permissionVariables?` here makes me worry.

Reply:
Could you elaborate pls?

Reply:
It's only the backend where we call these `customClaims`, but everywhere else on the UI and in our docs, we call these `permissionVariables`. Hence the name for the variable. Should I change it back to `customClaims`?